lib/events.js

```diff
@@ -41,6 +41,13 @@ EventEmitter.prototype._maxListeners = undefined;
 // added to it. This is a useful default which helps finding memory leaks.
 var defaultMaxListeners = 10;
 
+var errors;
+function lazyErrors() {
+  if (errors === undefined)
+    errors = require('internal/errors');
+  return errors;
+}
```

Comment:
Why do we need to lazily load this? Is this the pattern used everywhere? Should we?

Reply:
It is because events.js is loaded before all the other modules in the dependency tree (because of bootstrapping `process`. Lazy loading `errors` prevents `errors` from being loaded before events is loaded.

Reply:
I would still rather inline this instead of using the function form but it should not be a blocker.